styles.css

```diff
@@ -200,11 +200,157 @@ li {
 .card_content {
   padding: 1rem;
   background: #1a2236;
-  height: 210px;
+  height: 200px;
 }
 
 .card_image {
   height: auto;
   width: 486px;
   vertical-align: middle;
 }
+
+.skills_section {
+  display: grid;
+  grid-gap: 16px;
+  padding: 1rem;
+  padding-top: 100%;
+  padding-bottom: 20%;
+}
+
+.skills_section ul {
+  list-style-type: disc;
+}
+
+.skills_section li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+}
+
+.skills_section h3 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 22px;
+  line-height: 30px;
+}
+
+.language {
+  color: #fff;
+  width: 100%;
+  height: 289px;
+  background: #1c528a;
+  background-image: url("images/tag.png");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  justify-content: center;
+  padding-left: 20%;
+  display: flex;
+  flex-direction: column;
+  line-height: 27px;
+}
+
+.frameworks {
+  color: #fff;
+  width: 100%;
+  height: 289px;
+  background: #e2505c;
+  background-image: url("images/ic_framework.png");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  justify-content: center;
+  padding-left: 20%;
+  display: flex;
+  flex-direction: column;
+  line-height: 27px;
+}
+
+.skills {
+  color: #fff;
+  width: 100%;
+  height: 289px;
+  background: #febf52;
+  background-image: url("images/ic_skills.png");
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  justify-content: center;
+  padding-left: 20%;
+  display: flex;
+  flex-direction: column;
+  line-height: 27px;
+}
+
+.contact-page {
+  position: relative;
+  flex-direction: column;
+  width: 100%;
+  height: 800px;
+}
+
+.contact-page h2 {
+  padding-top: 10%;
+  white-space: nowrap;
+}
+
+.contact_section {
+  background: #f9f9fa;
+  display: flex;
+  position: relative;
+}
+
+form {
+  padding: 18px;
+}
+
+input[type=text],
+[type=email] {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #dfdfe0;
+  background: #fff;
+}
+
+textarea {
+  width: 100%;
+  height: 150px;
+  padding: 12px 20px;
+  box-sizing: border-box;
+  border: 1px solid #dfdfe0;
+  border-radius: 4px;
+  background: #fff;
+  font-size: 16px;
+  resize: none;
+  margin: 8px 0;
+  margin-bottom: 20px;
+}
+
+input[type=submit] {
+  background-color: #1a2236;
+  border: none;
+  color: white;
+  padding: 16px 32px;
+  border-radius: 4px;
+  text-decoration: none;
+  margin: 4px 2px;
+  cursor: pointer;
+  width: 490px;
+  height: 55px;
+}
+
+input[type=submit]:hover {
+  background-color: black;
+
+  /* SET COLOR IN BLACK */
+}
+
+.form_section {
+  display: flex;
+  position: relative;
+  padding-top: 50%;
+}
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
- [ ] You have a horizontal scroll at a screen size greater than `375px` on a mobile screen so please kindly fix this to your project responsive.

| Your Design |Figma Design |
| :---:   | :---: | 
|![Screenshot from 2022-12-06 23-43-34](https://user-images.githubusercontent.com/92303891/206049947-039d3cf7-29c0-46d2-aff6-4d3ff6627bec.png)| ![Screenshot from 2022-12-06 23-41-19](https://user-images.githubusercontent.com/92303891/206049967-c948bbc6-c455-429d-8ba6-8d8dab4a5523.png)|